e2e/local-e2e.sh

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash
```

Comment:
cc @Sylfrena, you might want to check these changes. FYI They look good to me.

Reply:
LGTM as well, definitely learnt some nifty scripting/ci stuff from the PR. :D